public/docs/ts/latest/guide/i18n.jade

```diff
@@ -86,12 +86,12 @@ a(id="angular2-i18n")
   Whatever approach we decide to use, we must update our templates to define which strings will be translated.
  
   
-a(id="i18n-directive")
+a(id="i18n-directives")
 .l-main-section
 :marked
-  ### i18n directive
+  ### i18n directives
```

Comment:
From a user perspective, under the context of the Angular docs, these look like directives. We should explain then that these are not directives if they truly aren't.

What is the difference between these and directives? Are they just meaningless attributes that are being used as a marker for `ng-xi18n` to pick up?